modules/tinymce/src/core/main/ts/api/html/Styles.ts

```diff
@@ -263,6 +265,16 @@ const Styles = (settings: StylesSettings = {}, schema?: Schema): Styles => {
               value = value.toLowerCase();
             }
 
+            // Convert RGB/RGBA colors to HEX
+            if (typeof settings.force_hex_color === 'string' && settings.force_hex_color !== 'off') {
+              RgbaColour.fromString(value).each((rgba) => {
+                //  Always convert or only convert if there will be no loss of information from the alpha channel
+                if (settings.force_hex_color === 'always' || rgba.alpha === 1) {
```

Comment:
What does `1` mean here? Might be worth making it a variable or adding a comment to avoid it being a magic number

Reply:
It's not a magic number; alpha is a fractional value where 1 is opaque and 0 is transparent. I don't know it's worth trying to explain this, we don't explain what `rgba.red` `rgba.green` or `rgba.blue` numbers mean (0 to 255). It's just something you need to know when dealing with colours, much like `hue` being a range from 0 to 360.